src/metabase/search/filter.clj

```diff
@@ -162,24 +162,24 @@
                               (search.config/column-with-model-alias model :created_at)
                               created-at))))
 
-
 ;; Last edited by filter
 
 (defn- joined-with-table?
   "Check if  the query have a join with `table`.
-  Note: this does a very shallow check by only checking the join-clause is already the same.
+  Note: this does a very shallow check by only checking if the join-clause is the same.
   Using the same table with a different alias will return false.
 
     (-> (sql.helpers/select :*)
-    (sql.helpers/from [:a])
-    (sql.helpers/join :b [:= :a.id :b.id])
-    (joined-with-table? :join :b))
+        (sql.helpers/from [:a])
+        (sql.helpers/join :b [:= :a.id :b.id])
+        (joined-with-table? :join :b))
 
     ;; => true"
   [query join-type table]
   (->> (get query join-type) (partition 2) (map first) (some #(= % table)) boolean))
 
-(defn- search-model->revision-model
+(defn search-model->revision-model
```

Comment:
This is not specific to search.filter anymore so it should be in a different namespace. Probably `search.config` near `model-to-db-model`

Reply:
good point, moved it to c056f07541